RELEASE/scripts/autoscend/combat/auto_combat.ash

```diff
@@ -256,6 +256,10 @@ string auto_combatHandler(int round, monster enemy, string text)
 		{
 			return "item " + $item[Filthy Poultice];
 		}
+		if(item_amount($item[red pixel potion]) > 0)
```

Comment:
Should add to the Ambidextrous Funkslinging block right above here as well.

Reply:
change made